pkg/controller/core/resourceflavor_controller.go

```diff
@@ -97,11 +106,22 @@ func (r *ResourceFlavorReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 	return ctrl.Result{}, nil
 }
 
+func (r *ResourceFlavorReconciler) AddUpdateWatcher(watchers ...ResourceFlavorUpdateWatcher) {
+	r.watchers = watchers
+}
+
+func (r *ResourceFlavorReconciler) notifyWatchers(rf *kueue.ResourceFlavor) {
+	for _, w := range r.watchers {
+		w.NotifyResourceFlavorUpdate(rf)
```

Comment:
nit:
We can make this function accept the old and new flavor objects. See `NotifyClusterQueueUpdate` below as an example.

This way, the ClusterQueue controller can filter only the creation and deletion of a RF. It doesn't really need to react to every ResourceFlavor update.

But I don't expect ResourceFlavor updates to be common, so this would be an optimization that can be left for the future.

Reply:
Sounds good. I would create an issue to raise this.

Reply:
#497